### ISSUE TITLE
Feature Request: Some kind of permission store rather than hard-coded moon privileges

### ISSUE THREAD

**Issue by tinnus-napbus** [urbit/urbit#4360]
**Problem**
Some people wanna hand out moons to friends/family but currently everything is `?>  (team:title our.bol src.bol)` so it's a big security risk and modifying every single app is annoying and beyond most people.

**Proposed Solution**
Create some kind of central permission store where you can explicitly allow/deny your moons and/or specific ships without having to modify source. Maybe make it so you can do it on an app-by-app basis

**Comment by philipcmonk** [comment]
I strongly agree with this.  It'd be nice if people could just hand out moons as easy non-sovereign "invites" for people to try it out without having to commit to buying a planet, storing keys, etc.  And that's not possible as long as `team:title` just checks if it's a moon.  Jael should just store a set of trusted moons, and `|moon` should take a trusted/untrusted flag.  Not too much work.
